segbot_sensors/CMakeLists.txt

```diff
@@ -82,3 +86,9 @@ if (CATKIN_ENABLE_TESTING)
   roslaunch_add_file_check(launch/kinect)
   roslaunch_add_file_check(launch/ptgrey)
 endif()
+
+add_executable(batteryDiagnostics src/segbot_sensors/battery_monitor.cpp)
+target_link_libraries(batteryDiagnostics ${catkin_LIBRARIES})
+
+add_executable(voltagePub src/segbot_sensors/fakeVoltPub.cpp)
+target_link_libraries(voltagePub ${catkin_LIBRARIES})
```

Comment:
We're typically using camel_case convention for files and executables. Could you:
1) Rename fakeVoltPub.cpp to fake_volt_pub.cpp.
2) I also prefer to have the binary names same as the file name, unless it's not possible for other reasons.
3) Once you've updated these, set the new targets to be installed as well. Add them at https://github.com/maxsvetlik/segbot/blob/segbot2/segbot_sensors/CMakeLists.txt#L58